script/Arcueid/Day 6/06_00_ARC06_3_1.txt

```diff
@@ -9,6 +9,7 @@ I'll go for a stroll out in the yard.
 [sha:3b6f818db70d1fbcb53c568a4f070f20a881784e]{
 -- Page 10011, Offset 14540.
 -- 屋敷に帰ってきてからこっち、庭園には出ていない。
+// change everything else to gardens
```

Comment:
Who volunteers. Though, honestly, I think garden and gardens can be used fairly interchangeably here.

Reply:
yeah I don't really think consistency matters that much here. I might change some if I edited files and noticed, but I'm also not that worried about it.